cloudbuild.yaml

```diff
@@ -20,7 +21,8 @@ steps:
       - -c
       - |
         gcloud auth configure-docker \
-        && make container-push
+        && apk add sed \
```

Comment:
the image has busybox linked as `sed` which does not work well with the `sed` command in our `Makefile` so install this explicitly